src/main/java/ai/rev/sentimentanalysis/models/SentimentAnalysisJob.java

```diff
@@ -0,0 +1,307 @@
+package ai.rev.sentimentanalysis.models;
+
+import ai.rev.speechtotext.models.asynchronous.RevAiJobType;
+import ai.rev.speechtotext.models.asynchronous.RevAiFailureType;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
```

Comment:
I saw recently that Slack swapped out gson for Moshi. We might consider the same in the future.